Automattic/sensei
# Add 'Not Started' to module progress preview

### Changes proposed in this Pull Request

* Add a default 'Not Started' option to module status preview, which hides the indicator
* Use radio boxes
* Share the setting component with lesson blocks. (Other changes related to lessons are in #3663)

### Testing instructions 

* Open a course with modules
* Check that module progress indicator is not visible
* Change the preview status in the block settings
* Check that the indicator is displayed with the correct status
* Check that changing the preview status for a lesson block also works



### Screenshot / Video

<img width="278" alt="image" src="https://user-images.githubusercontent.com/176949/95076954-f66bd580-0712-11eb-90e3-fad39e3cad55.png">
<img width="612" alt="image" src="https://user-images.githubusercontent.com/176949/95076980-008dd400-0713-11eb-8289-004fa4b7a500.png">
